_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -5224,10 +5226,8 @@
 "bWz" = (/obj/structure/disposalpipe/segment,/turf/simulated/floor/plasteel{dir = 1; icon_state = "whitebluecorner"; tag = "icon-whitebluecorner"},/area/medical/medbay2)
 "bWA" = (/obj/machinery/atmospherics/pipe/simple/hidden/supply,/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/turf/simulated/floor/plasteel{icon_state = "white"},/area/medical/medbay2)
 "bWB" = (/obj/structure/table/reinforced,/obj/item/weapon/wrench,/obj/item/weapon/crowbar,/obj/item/weapon/reagent_containers/spray/cleaner,/obj/machinery/power/apc{dir = 2; name = "south bump"; pixel_y = -24},/obj/structure/cable,/obj/machinery/door_control{desc = "A remote control switch for the medbay foyer."; id = "CloningDoor"; name = "Cloning Doors Control"; normaldoorcontrol = 1; pixel_x = 0; pixel_y = -36; range = 3},/turf/simulated/floor/plasteel{icon_state = "freezerfloor"},/area/medical/genetics_cloning)
-"bWC" = (/obj/machinery/computer/cloning,/turf/simulated/floor/plasteel{icon_state = "freezerfloor"},/area/medical/genetics_cloning)
-"bWD" = (/obj/machinery/dna_scannernew,/turf/simulated/floor/plasteel{icon_state = "freezerfloor"},/area/medical/genetics_cloning)
-"bWE" = (/obj/machinery/portable_atmospherics/canister/oxygen{name = "Canister: \[O2] (CRYO)"},/obj/structure/window/reinforced{dir = 8},/obj/machinery/door/window/eastright{dir = 1; icon_state = "left"; name = "Cryo Tank Storage"; req_access_txt = "0"; req_one_access_txt = "5;32"},/obj/machinery/atmospherics/unary/portables_connector{dir = 1},/turf/simulated/floor/plasteel{icon_state = "freezerfloor"},/area/medical/genetics_cloning)
```

Comment:
Replace it with the yellow canister from maintenance, with nothing in it, for looks.